db/seeds.rb

```diff
@@ -1,7 +1,12 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
+
+user = User.create(name: "mrpants", password: "yolo")
+
+5.times do
```

Comment:
If you install FactoryGirl you can use it in both seeds.rb _as well as_ your tests so you don't have to do convoluted eaching and looping (like the below).